docs/using-est-with-certd.md

```diff
@@ -0,0 +1,187 @@
+# Using EST with Certificates Service
+
+This document covers how to configure Certificates Service to issue certificates over EST.
+
+## Prerequisites
+
+- Certificates Service and Keys Service configured and running
+- A working EST server that processes enrollment requests
+
+## Configuration Options
+
+EST certificate issuance is configured in Certificates Service's config.toml, which by default is `/etc/aziot/certd/config.toml` or a file in `/etc/aziot/certd/config.d/`.
```

Comment:
After a default installation I only see a `/etc/aziot/certd/config.toml.default` file - no `config.toml`.  Are you referring to the config.toml.default or would the guidance be that someone can simply create a config.toml and certd will know to pick it up by default?  It's not clear here so we should clarify.

Reply:
I think you need to rename/copy config.toml.default to config.toml, and edit it as described here. Maybe we should specify that here?

Reply:
The expectation is for the user to just create the files. I added a section.